commands/utils/quick-pick-azure.ts

```diff
@@ -128,36 +130,30 @@ export async function quickPickResourceGroup(canCreateNew?: boolean, subscriptio
 export async function confirmUserIntent(yesOrNoPrompt: string): Promise<boolean> {
     let opt: vscode.InputBoxOptions = {
         ignoreFocusOut: true,
-        placeHolder: 'No',
+        placeHolder: 'Yes',
```

Comment:
After playing with it, I'm suggesting a change:
  placeHolder: 'Enter "Yes"',

Without that, it looks too much like a default value.